.github/workflows/e2e_libvirt.yaml

```diff
@@ -84,7 +88,8 @@ jobs:
 
       - name: Config Libvirt
         run: |
-          ./libvirt/config_libvirt.sh
+          echo "secure_comms=\"${{ inputs.secure_comms }}\""
+          ./libvirt/config_libvirt.sh ${{ inputs.secure_comms }}
```

Comment:
What is the end purpose of this change and adding secure comms as a parameter of config_libvirt.sh?

Reply:
Different tests in the matrix require different parameters to be set in libvirt.properties
See for example: https://github.com/confidential-containers/cloud-api-adaptor/pull/2089/files#diff-5612979f81fb0700a41c09f8fc0c55c15a264590115690474d10b6b46d40c2ffR136

Reply:
okay, I don't love it, but I guess the properties output has become core to config_libvirt.sh now. Did you consider driving it with an env rather than parameter?

Reply:
Moved to a TEST_E2E_SECURE_COMMS env variable